OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -36,6 +36,9 @@ public class WithIdleOverlayInfo : UpgradableTraitInfo, IRenderActorPreviewSprit
 
 		public readonly bool PauseOnLowPower = false;
 
+		[Desc("Optional decoration element which doesn't cast shadows, shouldn't get overlayed and won't change palettes on upgrade.")]
```

Comment:
That line marks the color overlay generated by the trait as a decoration.  It's exactly the same as the low power shadow overlay, but generic.